client/my-sites/plugins/plugin-details-header/index.jsx

```diff
@@ -18,11 +18,16 @@ const PluginDetailsHeader = ( { plugin, isPlaceholder } ) => {
 
 	return (
 		<>
-			<div className="plugin-details-header__tags">{ tags }</div>
 			<div className="plugin-details-header__container">
-				<div className="plugin-details-header__name">{ plugin.name }</div>
-				<div className="plugin-details-header__description">
-					{ preventWidows( plugin.short_description || plugin.description ) }
+				<div className="plugin-details-header__tags">{ tags }</div>
+				<div className="plugin-details-header__main-info">
+					<img className="plugin-details-header__icon" src={ plugin.icon } alt="Plugin Icon" />
```

Comment:
I love that you included an alt text for the icon, thank you! In this instance however I wonder if it should be [considered a decorative image](https://www.w3.org/WAI/tutorials/images/decorative/) where we could leave the attribute empty?